_studio/mfx_lib/shared/include/mfx_brc_common.h

```diff
@@ -196,6 +196,7 @@ class cBRCParams
     mfxU16 gopPicSize;
     mfxU16 gopRefDist;
     bool   bPyr;
+    bool   bFieldMode;
```

Comment:
In fact it is specific for HEVC only joining of fields to frames. May be call it bHevcFields or bJoinFields?

Reply:
I think, it is OK. for H264 SW BRC works in progressive mode for both interlace and progressive